n2kMapper.js

```diff
@@ -36,6 +43,15 @@ var toDelta = function(n2k) {
   }
 }
 
+function callFunction(aFunction, n2k, state) {
```

Comment:
I don't think this mechanism is needed: you can call a JS function with extra arguments and it will silently ignore the extra ones.

![image](https://user-images.githubusercontent.com/1049678/27997080-3f6e0b50-64f9-11e7-8f8c-8acb4bbe1ae1.png)